start.py

```diff
@@ -27,7 +27,7 @@
 logger.setLevel(buildpackutil.get_buildpack_loglevel())
 
 
-logger.info('Started Mendix Cloud Foundry Buildpack v2.0.3')
+logger.info('Started Mendix Cloud Foundry Buildpack v2.1.2')
```

Comment:
Where are versions `v2.1.0` and `v2.1.1` ?

Reply:
2.1 was actually already released (without bumping the version number). Adding a new change for DD is 2.1.1 which makes this 2.1.2.